src/material/schematics/ng-update/upgrade-rules/package-imports-v8/material-symbols.json

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
Thinking about it. Since the type checker is not available for 9.x releases, and this migration runs for target v9, would it be better to run against the latest `9.x` release (where we will introduce the migration), and then only use that map when the migration rule targets `9.x`?

The way it is right now, it could be that we miss a few new symbols that were added. or did you confirm there is no difference in terms of exposed symbols (_what are the obscure symbols_?)

Reply:
I see, since we would support both v8 and v9, technically we would need two mappings, but if you say that the v8 symbols are a superset of the v9 symbols, then we could just build one for v9 and use that?

Reply:
I think generating from `8.2.x` is fine. I'm not concerned about the difference of a few edge-case symbols